src/server/passive_tran_server.cpp

```diff
@@ -56,6 +56,12 @@ passive_tran_server::connection_handler::get_request_handlers ()
   return handlers_map;
 }
 
+log_lsa
+passive_tran_server::connection_handler::get_saved_lsa () const
+{
+  assert (false); // Only used in active transaction server
```

Comment:
I have no idea but defining `get_saved_lsa()` in tran_server to access it through `tran_server::connection_handler` in `active_tran_server::compute_consensus_lsa()`. Please tell me if you have a better idea.

Reply:
It is ok like this.
I understand it is a bit contrived but I have no better idea ... (It is the price to pay for not using a `dynamic_cast`)

Reply:
However, you still need to return a `NULL_LSA`.
If you look into the compile output from gcc you'll see a warning for not returning a value.

Reply:
Thank you for the advice.